vignettes/slowstart.Rmd

```diff
@@ -74,31 +67,19 @@ Data for Level 2 units can be returned by using the `level = "2"` argument. The
 
 An example of a country with Level 2 units is Belgium, where Level 2 units are Belgian provinces:
 
-```{r}
-get_regional_data("Belgium", level = "2")
+```{r, eval=TRUE, message=FALSE}
+get_regional_data(country = "france", level = "2")
```

Comment:
change in country and no table format. It would be great for the vignette to look as before (i.e clean code with no output and then the table generated using `knitr::kable` from an invisible block.